phoenix-hive-base/src/it/java/org/apache/phoenix/hive/HivePhoenixStoreIT.java

```diff
@@ -362,4 +362,60 @@ public void testTimestampPredicate() throws Exception {
         createFile(sb.toString(), fullPath);
         runTest(testName, fullPath);
     }
+
+    @Test
+    //This SHOULD test for PHOENIX-6524 when all children of an AND expression are pushed down,
+    //and we were returning an AND expression without children to Hive.
+    //However, this bug didn't trigger in this test, only on an actual deployment
```

Comment:
I've ran the newly created test without the other changes and it failed for me, despite this comment.
With the rest of the changes it passes.

Reply:
Yes, it fails with MR, but not with TEZ.
I have no clue why. 

In any case, I've updated the comment, and stripped the superfloud columns now.